apps/a_dndtoggle/a_dndtoggle.app.js

```diff
@@ -6,11 +6,16 @@ let current = 0|bSettings.quiet;
 //1 alarms
 //2 silent
 
+const dndSettings = Object.assign(
+		require('Storage').readJSON("a_dndtoggle.default.json", true) || {},
+		require('Storage').readJSON("a_dndtoggle.settings.json", true) || {}
+  );
+
 console.log("old: " + current);
 
 switch (current) {
 	case 0:
-		bSettings.quiet = 2;
+		bSettings.quiet = dndSettings.mode;
```

Comment:
I'll leave it up to you, but we could save having an extra file (and the extra time to read it) and apply the default here and in settings, like so:

```suggestion
		bSettings.quiet = dndSettings.mode || 2;
```